### ISSUE TITLE
Could you add some instructions on how to fine-tune the pretrained model in both step 1 and step 2?

### ISSUE THREAD

**Issue by jocskywalker** [GATECH-EIC/ViTCoD#5]
Hi,
Could you add some instructions on how to fine-tune the pretrained model in both step 1 and step 2? In step 1, I tried the config below: python main.py --model deit_tiny_patch16_224      --data-path /mnt//ViTCoD-main/data/CLS-LOC     --lr 1e-5    --weight-decay 1e-8     --epochs 100   --resume exp_hr/svd/deit_tiny/deit_tiny_patch16_224-a1311bcf.pth  --output_dir exp_hr/lowrank_sparse/deit_tiny/deit_tiny_info50     --svd_type 'mix_head_fc_qk'      --need_weight , but get a very low accuracy in imagenet1k, the pretrained model is download from the DeiT official Lib. And should I use the gen_mask to create sparse mask before the step 2 training? 

Thanks in advance
![6ef837dbca2e16b6f460503ddfbfebe0](https://github.com/user-attachments/assets/3052fe2a-9b48-41fc-a51d-a6a7c029d2bf)
